src/local/butler/weights.py

```diff
@@ -0,0 +1,258 @@
+  if entry_type == EntryType.FUZZER_JOB:
+    _dump_fuzzer_jobs()
+  elif entry_type == EntryType.FUZZER_JOBS:
+    _dump_fuzzer_jobs_batches()
+
+
+def _fuzzer_job_matches(
+    fuzzer_job: data_types.FuzzerJob,
+    fuzzers: Optional[Sequence[str]],
+    jobs: Optional[Sequence[str]],
+) -> bool:
+  """Returns whether the given FuzzerJob matches the given optional filters."""
+  if fuzzers and fuzzer_job.fuzzer not in fuzzers:
+    return False
+
+  if jobs and fuzzer_job.job not in jobs:
+    return False
+
+  return True
+
+
+def _print_stats(fuzzer_jobs: List[data_types.FuzzerJob],
+                 total_weight: float) -> None:
+  """Helper for `_aggregate_fuzzer_jobs()`."""
+  weight = _sum_weights(fuzzer_jobs)
+  probability = weight / total_weight
+
+  print(f'  Count: {len(fuzzer_jobs)}')
+  print(f'  Total weight: {weight}')
+  print(f'  Total probability: {_display_prob(probability)}')
+
+  # New in Python 3.8. We appease the linter by disabling `no-member` below.
+  if not hasattr(statistics, 'quantiles'):
+    return
+
+  # `quantiles()` returns n-1 cut points between n quantiles.
+  # `weight_deciles[0]` separates the first from the second decile, i.e. it is
+  # the 10% percentile value. `weight_deciles[i]` is the (i+1)*10-th.
+  weight_deciles = statistics.quantiles(_iter_weights(fuzzer_jobs), n=10)  # pylint: disable=no-member
+  weight_median = weight_deciles[4]
+  weight_90p = weight_deciles[8]
+
+  prob_median = weight_median / total_weight
+  prob_90p = weight_90p / total_weight
+
+  print(f'  Median weight: {weight_median}')
+  print(f'  Median probability: {_display_prob(prob_median)}')
+  print(f'  90th percentile weight: {weight_90p}')
+  print(f'  90th percentile probability: {_display_prob(prob_90p)}')
+
+
+def _aggregate_fuzzer_jobs(
+    platform: str,
+    fuzzers: Optional[Sequence[str]] = None,
+    jobs: Optional[Sequence[str]] = None,
+) -> None:
+  """Aggregates statistics for matching and non-matching FuzzerJob entries."""
+  fuzzer_jobs = list(_query_fuzzer_jobs(platforms=[platform]))
+  total_weight = _sum_weights(fuzzer_jobs)
+
+  matches = []
+  others = []
+  for fuzzer_job in fuzzer_jobs:
+    if _fuzzer_job_matches(fuzzer_job, fuzzers, jobs):
+      matches.append(fuzzer_job)
+    else:
+      others.append(fuzzer_job)
+
+  print('Matching FuzzerJob entries:')
+  _print_stats(matches, total_weight)
+  print('Other FuzzerJob entries:')
+  _print_stats(others, total_weight)
+
+
+def execute(args) -> None:
+  """Entrypoint from butler.py."""
+  os.environ['CONFIG_DIR_OVERRIDE'] = args.config_dir
+  local_config.ProjectConfig().set_environment()
+
+  with ndb_init.context():
+    cmd = args.weights_command
+    if cmd == 'platforms':
+      list_platforms()
+    elif cmd == 'dump':
+      _dump_entries(EntryType(args.type))
+    elif cmd == 'list':
+      _list_fuzzer_jobs(
+          _query_fuzzer_jobs(
+              platforms=args.platforms, fuzzers=args.fuzzers, jobs=args.jobs))
```

Comment:
Maybe it would be useful to ignore the case of the argument provided.

Reply:
Great idea! Fixed.